vnext/Microsoft.ReactNative/Modules/PaperUIManagerModule.h

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
can we call this file something else? Codenames don't necessarily register well :)

Reply:
It implements the native side of:
https://github.com/facebook/react-native/blob/master/Libraries/ReactNative/PaperUIManager.js